core/jvm/src/test/scala/com/softwaremill/sttp/testing/TestHttpServer.scala

```diff
@@ -59,158 +62,171 @@ private class HttpServer(port: Int) extends AutoCloseable {
   private implicit val actorSystem: ActorSystem = ActorSystem("sttp-test-server")
   private implicit val materializer: ActorMaterializer = ActorMaterializer()
 
+  private val corsSettings = CorsSettings.defaultSettings
+    .withExposedHeaders(List("Server", "Date", "Cache-Control", "Content-Length", "Content-Type", "WWW-Authenticate"))
+
   private def paramsToString(m: Map[String, String]): String =
     m.toList.sortBy(_._1).map(p => s"${p._1}=${p._2}").mkString(" ")
 
   private val textFile = new java.io.File(getClass.getResource("/textfile.txt").getFile)
   private val binaryFile = new java.io.File(getClass.getResource("/binaryfile.jpg").getFile)
   private val textWithSpecialCharacters = "Żółć!"
 
-  val serverRoutes: Route =
-    pathPrefix("echo") {
-      pathPrefix("form_params") {
-        formFieldMap { params =>
-          path("as_string") {
-            complete(paramsToString(params))
-          } ~
-            path("as_params") {
-              complete(FormData(params))
+  val serverRoutes: Route = {
+    handleRejections(CorsDirectives.corsRejectionHandler) {
+      cors(corsSettings) {
```

Comment:
I assume that nothing changed here except for the cors directive at the top?

Reply:
Yeah, I'll put it in a variable so the diff looks nicer.